phoenicis-apps/src/main/java/org/phoenicis/apps/MergeableApplicationsSource.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * 
+ */
+package org.phoenicis.apps;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.function.Function;
+
+import org.phoenicis.apps.dto.ApplicationDTO;
+import org.phoenicis.apps.dto.CategoryDTO;
+import org.phoenicis.apps.dto.ResourceDTO;
+import org.phoenicis.apps.dto.ScriptDTO;
+
+public interface MergeableApplicationsSource extends ApplicationsSource {
+	public default CategoryDTO mergeCategories(CategoryDTO leftCategory, CategoryDTO rightCategory) {
```

Comment:
What do you mean with "reformat" here? Do you mean I'm using the wrong formatter settings? Or that this shouldn't be an interface but a class?